coral/src/app/features/approvals/acls/AclApprovals.tsx

```diff
@@ -40,7 +44,11 @@ interface AclRequestTableRow {
   aclType: AclRequest["aclType"];
   username: string;
   requesttimestring: string;
-  requestStatus: "CREATED" | "DELETED" | "DECLINED" | "APPROVED" | "ALL" | "-";
+  // `requestStatus` is always defined from backend
+  // but api definition says it can be undefined
+  // the empty string is used to make ts compiler
+  // happy :D
+  requestStatus: RequestStatus | "";
```

Comment:
In `AclRequest`,  the `req_no` is not required - I think it's the same as with `SchemaRequest`, the backend uses the same Model for the "create new" call as "approve". When creating, there is of course no `req_no` yet, but the schema-request for approver never returns a `SchemaRequest` with an undefined `req_no`.

For schema requests, I designed the types in frontend accordingly.

Since updating the types for `AclRequest` is a bigger change and needs sync, I added this additional handling of setting the `requestStatus` here to possible be `""` and handling that in the rows, even so it will never be undefined.

We can update that later - either on our end, or, even better, on api side! :D

Reply:
I wonder if we could do something like this in the meantime :o It would actually be true to what the API expects! Even if TS is a little overruled. It also means we only have one spot where we are doing some "lying" about the API, instead of having all the lttle spots in the UI code where we make provisions for a possible `undefined` which actually doesn't exist.

```ts
type AclRequest = ResolveIntersectionTypes<
  KlawApiModel<"aclRequest"> & {
    req_no: RequestStatus;
  }
>;
```

I don't expect that this model will be fixed soon if it's so strictly tied to two different UI element in the Angular app :o

Reply:
We would have to create two types, like with Schemas, because `AclRequest` is also used for creating an ACL, where status does not make sense to be required. That's why I did it like that for now, to not do too big changes here. But we should add it as a follow up housekeeping ticket!

Reply:
Ah I see, of course. 

I would for sure create an`ApproverAclRequest` though, then we get this, which makes sense to me

```
type ApproverAclRequest = ResolveIntersectionTypes<
  KlawApiModel<"aclRequest"> & {
    req_no: number;
  }
>;

type AclRequestsForApprover = ResolveIntersectionTypes<
  Paginated<ApproverAclRequest[]>
>;
```

Reply:
ok! I added a housekeeping ticket for that one #717  

If the rest is ok... you could approve and merge? 😇 🙏 🥺